test/unit/Github/CreateChangelogTextTest.php

```diff
@@ -10,12 +10,14 @@
 use Laminas\AutomaticReleases\Github\CreateReleaseTextThroughChangelog;
 use Laminas\AutomaticReleases\Github\GenerateChangelog;
 use Laminas\AutomaticReleases\Github\Value\RepositoryName;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
 final class CreateChangelogTextTest extends TestCase
 {
     public function testGeneratedReleaseText(): void
     {
+        /** @psalm-var GenerateChangelog&MockObject $generateChangelog */
```

Comment:
This inline type shouldn't be needed: phpunit mocking utilities already have the correct annotations

Reply:
This was in part for intelephense code completion.

Reply:
Doesn't intelephense read something like `.phpstorm.meta.php`?